package.json

```diff
@@ -10,9 +10,11 @@
   },
   "main": "index.js",
   "scripts": {
+    "build": "npm run clean:css",
```

Comment:
since there is already a `build` in the latest `master`, maybe change to:
```json
"scripts": {
  "build": "npm run build:index && npm run build:css",
  "build:index": "node bin/index.js README.md >> dist/index.html",
  "build:css": "..."
}
```